Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -339,6 +350,17 @@ - (void)URLSession:(__attribute__((unused)) NSURLSession *)session
                                      forDownloadTask:downloadTask];
         break;
       }
+      case NYPLMyBooksDownloadRightsManagementAxis: {
+#if defined(AXIS)
```

Comment:
This will be taken care of after the code for dealing with Axis books is merged in (in the next PR). As of now, in the context of this PR, SimplyE/OpenEbooks have no idea who/what Axis is or how it downloads/opens books.